composer.json

```diff
@@ -22,5 +22,8 @@
       "Cors\\Test\\": "tests",
       "Cake\\Test\\": "./vendor/cakephp/cakephp/tests"
     }
+  },
+  "scripts": {
+    "test": "phpunit"
```

Comment:
This is not needed for the CI part, only a convenience method so you can just execute `composer test`